pam/integration-tests/testdata/golden/TestSSHAuthenticate/Authenticate_user_locks_and_unlocks_it_with_shared_sshd

```diff
@@ -2,50 +2,51 @@
 == Provider selection ==
   1. local
   2. ExampleBroker
-(user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-on-shared-sshd@localhost) Choose your provider:
+(user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-with-shared-sshd@localhost) Choose your provider:
 >
 ────────────────────────────────────────────────────────────────────────────────
 > ssh ${AUTHD_PAM_SSH_USER}@localhost ${AUTHD_PAM_SSH_ARGS}
 == Provider selection ==
   1. local
   2. ExampleBroker
-(user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-on-shared-sshd@localhost) Choose your provider:
+(user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-with-shared-sshd@localhost) Choose your provider:
 > 2
 ────────────────────────────────────────────────────────────────────────────────
 > ssh ${AUTHD_PAM_SSH_USER}@localhost ${AUTHD_PAM_SSH_ARGS}
 == Provider selection ==
   1. local
   2. ExampleBroker
-(user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-on-shared-sshd@localhost) Choose your provider:
+(user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-with-shared-sshd@localhost) Choose your provider:
 > 2
 == Password authentication ==
 Enter 'r' to cancel the request and go back to select the authentication method
-(user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-on-shared-sshd@localhost) Gimme your password:
+(user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-with-shared-sshd@localhost) Gimme your password:
 >
 ────────────────────────────────────────────────────────────────────────────────
 > ssh ${AUTHD_PAM_SSH_USER}@localhost ${AUTHD_PAM_SSH_ARGS}
 == Provider selection ==
   1. local
   2. ExampleBroker
-(user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-on-shared-sshd@localhost) Choose your provider:
+(user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-with-shared-sshd@localhost) Choose your provider:
 > 2
 == Password authentication ==
 Enter 'r' to cancel the request and go back to select the authentication method
-(user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-on-shared-sshd@localhost) Gimme your password:
+(user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-with-shared-sshd@localhost) Gimme your password:
 >
-PAM Authenticate() finished for user 'user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-on-shared-sshd'
-PAM AcctMgmt() finished for user 'user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-on-shared-sshd'
+PAM Authenticate() finished for user 'user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-with-shared-sshd'
+PAM AcctMgmt() finished for user 'user-integration-pre-check-ssh-authenticate-user-locks-and-unlocks-it-with-shared-sshd'
+Could not chdir to home directory /tmp/TestSSHAuthenticate2114809203/011/authd-test-user-sshd-accept-all: No such file or directory
```

Comment:
@3v1n0 do you understand what's causing this line to be in the output of all `TestSSHAuthenticate` tests when I run them locally but not in the CI? It's very annoying because 1. the tests always fail locally due to diff in the golden files and 2. when I update the golden files locally without removing these lines manually, the tests fail in the CI

Reply:
Is it related to `pam_mkhomedir`?

Reply:
Yes and the fact that apparently it does not does its job, which I wanted to actually check in tests via https://github.com/ubuntu/authd/pull/979